chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.css

```diff
@@ -166,3 +167,60 @@
   padding: var(--leo-spacing-l);
   box-sizing: border-box;
 }
+
+.enable-tab-focus-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
do we need all these `width: 100%`'s? Maybe we should use `align-items: stretch` instead (I think width is overridden by the `flex-align`)

Reply:
Removed `width: 100%` and changed them to `align-items: stretch`